Dockerfile

```diff
@@ -29,27 +34,30 @@ COPY /jitsi_external_api.min.js /webapp/jitsi_external_api.min.js
 
 # Configuring certificate while SSL authentication is true
 # Configure yarn to use https proxy - https://stackoverflow.com/questions/51508364/yarn-there-appears-to-be-trouble-with-your-network-connection-retrying
-RUN yarn config set https-proxy http://ACTGOV%5C[Your first name]%20[Your Surname]:[Only insert your password (URL encoded) when required]@proxy.test.act.gov.au:9090
-RUN yarn config set strict-ssl true
-RUN yarn config set cafile /opt/PRDPKICA.crt
+ARG HTTPS_PROXY
+RUN yarn config set https-proxy ${HTTPS_PROXY} \
+ && yarn config set strict-ssl true \
+ && yarn config set cafile /opt/PRDPKICA.crt
 
 # Not mandatory when working with pegacorn sdk branches that were pulled locally
 # This explains why the ARG variables are commented out
 # RUN dos2unix /src/scripts/docker-link-repos.sh && bash /src/scripts/docker-link-repos.sh
 
-RUN cd pegacorn-communicate-matrix-js-sdk
-RUN yarn link
-RUN yarn --network-timeout 100000 install
-RUN cd ../
-
-RUN cd pegacorn-communicate-matrix-react-sdk
-#RUN yarn link
-RUN yarn link element-web
-RUN yarn --network-timeout 100000 install
-RUN cd ../
-
-RUN yarn --network-timeout 100000 install
-RUN yarn build
+WORKDIR /src/pegacorn-communicate-matrix-js-sdk
+RUN yarn link \
+ && yarn --network-timeout 100000 install
+ 
+WORKDIR /src/pegacorn-communicate-matrix-react-sdk
+RUN yarn link \
+ && yarn link matrix-js-sdk \
+ && yarn --network-timeout 100000 install \
```

Comment:
yarn reskindex can turn lethal as well as I had some issues when I ran this command, app would not build so i had to re unlink and link project . Yarn seems to pick the old symlinking from cached memory and try to read old linked project. 
Also, just so you know that yarn reskindex is not required if there is no changes in react-sdk project, if new page is added then that needs to be added in index.js which triggers need for reskindex command otherwise line of code changes in existing file wont need this command according to react-sdk readme documentation.I had issue running this command locally at one point to reflect react sdk changes in app-web, i had to remove npm module, then do re linking of project then install npm then build project but yeah , if reskindex is not causing issue for you, we can experiment this in our build as you have suggested in modification. Need of reskindex has been described as part of building out new component added in directory so I presume locally running this command might add directory into index.js in react sdk ultimately leading to not requiring this in dockerbuild maybe.

Reply:
This is done in the element docker file build (via the scripts/docker-link-repos.sh) so I just wanted to minimise any changes ... seems have to worked fine in the docker build, which I have tested on my DEV Azure Workstation.